tests/pos-with-compiler-cc/backend/jvm/BCodeIdiomatic.scala

```diff
@@ -55,7 +55,8 @@ trait BCodeIdiomatic extends Pure {
     case "18" => asm.Opcodes.V18
     case "19" => asm.Opcodes.V19
     case "20" => asm.Opcodes.V20
-    case "21" => asm.Opcodes.V21*/
+    case "21" => asm.Opcodes.V21
+    case "22" => asm.Opcodes.V22*/
```

Comment:
Why are those commented out?

Reply:
Oh, I see. This is in the `pos-with-compiler-cc` tests. We do not need to update this code. It is fine to keep this update, but it is not necessary.